Documentation/tutorial/docfx.exe_user_manual.md

```diff
@@ -90,7 +90,7 @@ docfx build [-o:<output_path>] [-t:<template folder>]
 ```
 `docfx build` generates documentation for current folder.
 
-If `toc.yml` or `toc.md` is found in current folder, it will be rendered as the top level TABLE-OF-CONTENT. As in website, it will be rendered as the top navigation bar.
+If `toc.yml` or `toc.md` is found in current folder, it will be rendered as the top level TABLE-OF-CONTENT. As in website, it will be rendered as the top navigation bar. Relative paths are relative to docfx.exe.
```

Comment:
What "Relative paths" here do you mean? If they are the `href` in `toc.yml` or `toc.md`, they are expected to be relative to the current TOC, instead of `docfx.exe`.